CONTRIBUTING.md

```diff
@@ -16,6 +16,10 @@
   - [React Component Requirements](#react-component-requirements)
   - [Code Consistency](#code-consistency)
   - [Code Contribution Guidelines](#code-contribution-guidelines)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+  - [How do I become a maintainer?](how-do-i-become-a-maintainer?)
```

Comment:
maintainer or committer? there might be a slight difference between the two

Reply:
@ohadlevy yeah I noticed this discrepancy between Foreman and PF. For PatternFly, each repo has two sets of teams, "Contributors" and "Maintainers". Therefore I decided to keep consistent with our current definitions as seen on our teams page: https://github.com/orgs/patternfly/teams

WDYT?